indico/modules/events/contributions/controllers.py

```diff
@@ -121,3 +126,27 @@ def _process(self):
                        "{count} contributions have been deleted.", deleted_count)
               .format(count=deleted_count), 'success')
         return jsonify_data(html=_render_contribution_list(self.event_new))
+
+
+class RHContributionPersonList(RHManageContributionsActionsBase):
+    """List of persons in the contribution"""
+
+    def _process(self):
```

Comment:
same code as for session persons besides `Contribution.id.in_(contribution_ids)`?